sdk/eventhub/Azure.Messaging.EventHubs/tests/Primitives/EventProcessorTests.StartStop.cs

```diff
@@ -1286,71 +1236,6 @@ public async Task StopProcessingLogsErrorDuringShutdown(bool async)
                 Times.Once);
         }
 
-        /// <summary>
-        ///   Verifies functionality of the <see cref="EventProcessor{TPartition}.StopProcessing" />
-        ///   method.
-        /// </summary>
-        ///
-        [Test]
-        [TestCase(true)]
-        [TestCase(false)]
-        public async Task StopProcessingLogsFaultedTaskDuringShutdown(bool async)
```

Comment:
The task now logs itself, so there's no reason for `StopProcessingAsync` to do it as well.